### PR TITLE
Aks ci

### PR DESCRIPTION
**Pull request by uek3-cdc**

### CODE REVIEW

File: .github/workflows/graphql-service-ci.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        with:
+          go-version: 1.22.0
+      - name: Vet
+        run: go vet ./...
+      - name: Run Tests
+        run: go test -coverprofile=c.out -coverpkg=./... ./...
+      - name: Show coverage
+        run: go tool cover -func=c.out
+      - name: Build Check
+        run: go build -o pstatus-graphql-ktor ./cmd/main.go
```
Comment (mkrystof) [inline .github/workflows/graphql-service-ci.yml:31]: The unit tests are not built in go, we use kotlin.  The run step should be something like `gradle test`.

File: .github/workflows/remote-cd-trigger-template.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        description: "Workflow yml file that should be triggered."
+        required: true
+      REF: 
+        type: string
+        description: "Git tag or branch the workflow is running on."
+        required: false
+        default: 'main'
+
+jobs:
+  invoke-cd-trigger:
```
Comment (mkrystof) [inline .github/workflows/remote-cd-trigger-template.yml:16]: Seems like there is a step missing to build and push the image to CDC ImageHub.  The step to do both the build and publish to ImageHub is simply `gradle jib`.  All that is required is the two ENV vars set for the image hub username and password.

**Review by mkrystof — DISMISSED** [review]

File: .github/workflows/graphql-service-ci.yml (hunk 1 above)
Comment (uek3-cdc) [reply .github/workflows/graphql-service-ci.yml]: gradle test  step added

File: .github/workflows/remote-cd-trigger-template.yml (hunk 2 above)
Comment (uek3-cdc) [reply .github/workflows/remote-cd-trigger-template.yml:16]: adding the imagehub environment variables at cdc gov and cdc ent repos and will see how this gradle jib works. was not sure how to include this at moment..

**Review by srinikovi20 — APPROVED** [review]

**Review by srinikovi20 — APPROVED** [review]

**Review by mkrystof — APPROVED** [review]

**Pull request merged**